pkg/rest/rest.go

```diff
@@ -0,0 +1,141 @@
+package rest
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/crc-org/vfkit/pkg/rest/define"
+	"github.com/gorilla/mux"
```

Comment:
For the record (we discussed this privately), https://github.com/gorilla/mux says "The Gorilla project has been archived, and is no longer under active maintainenance", we'll need to move to some other project.